### ISSUE TITLE
Problem with hiding widgets in theme-options.php

### ISSUE THREAD

**Issue by roikles** [roikles/Flexbones#59]
Fix ---

Replace:

```
add_action('customize_register', 'remove_customize_bloat');

function remove_customize_bloat($wp_customize) {
    $wp_customize->remove_panel('widgets');
    $wp_customize->remove_section( 'static_front_page' );
    $wp_customize->remove_section( 'title_tagline' );
}
```

With:

```
function wpdocs_remove_widgets_panel( $components ) {
    $i = array_search( 'widgets', $components );
    if ( false !== $i ) {
        unset( $components[ $i ] );
    }
    return $components;
}
add_filter( 'customize_loaded_components', 'wpdocs_remove_widgets_panel' );
```

**Comment by DrizzlyOwl** [comment]
Good idea - friendly reminder to rename the function prefix when including in the theme.

**Comment by roikles** [comment]
GG. I just copied and pasta.

**Comment by roikles** [comment]
Fixed in 87212c8
